code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4197,3 +4195,89 @@
 /obj/item/weapon/reagent_containers/food/snacks/frog_leg/New()
 	..()
 	reagents.add_reagent(NUTRIMENT, 6)
+
+
+/obj/item/weapon/reagent_containers/food/snacks/pie/welcomepie
+	name = "friendship pie"
+	desc = "Offered as a gesture of Vox goodwill." //"Goodwill"
+	icon_state = "welcomepie"
+	bitesize = 4
+	New()
+		..()
+		reagents.add_reagent(SACID,6)
+		reagents.add_reagent(NUTRIMENT,2)
+
+/obj/item/weapon/reagent_containers/food/snacks/zhulongcaofan
+	name = "zhu l�ng cao f�n"
```

Comment:
If I remember correctly, they appear in the chat, but not on mouseover or other locations.
Or maybe it was the other way around.

I remember from when some of the golem names had accented characters, that the characters would only appear in one place.